### ISSUE TITLE
Unable to find module called "GridLinesMgr"

### ISSUE THREAD

**Issue by 10110111** [Stellarium/stellarium#3324]
Since some time I started noticing this message in the log:
```
Unable to find module called "GridLinesMgr"
```
I didn't try to check why it happens, but it doesn't look normal. I hope you can reproduce: it simply appears every time during startup of Stellarium. [log.txt](https://github.com/Stellarium/stellarium/files/12021751/log.txt)

**Comment by gzotti** [comment]
Yes, I have also seen it, without tracing it down. Some time ago I have added an option to suppress this message, intended for checking for optional plugins.  But a core module should be available all times...
